### PR TITLE
update weave add-on to use patched images

### PR DESCRIPTION
**Pull request by garcialuis**
#### What type of PR is this?
type::chore

#### What this PR does / why we need it:
Updates images for weave add-on versions 2.6.5 and 2.8.1. New images are patched for CVEs.

#### Which issue(s) this PR fixes:

Fixes #
[SC-49336](https://app.shortcut.com/replicated/story/49336/weave-2-8-1-cves)
#### Does this PR introduce a user-facing change?

```release-note
Fixes CVEs for weave 2.8.1 plugin
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

**Review by emosbaugh — CHANGES_REQUESTED** [review]
you will need to add EXEC_IMAGE env var to pods like so https://github.com/replicatedhq/replicated-installer/blob/06a2cb9fc32731386e23afe87456b23386565610/install_scripts/templates/kubernetes/yml-generate.sh#L966-L967

can you find out if we have the ability to run airgap tests on pr from @laverya ? if so can you add? i would think we would have caught this without my review

File: scripts/tasks.sh [hunk 1]
```diff
@@ -396,8 +396,14 @@ function get_weave_version() {
     if [ -z "$weave_version" ]; then
         if [ -n "$DOCKER_VERSION" ]; then
             weave_version=$(docker image ls | grep weaveworks/weave-npc | awk '{ print $2 }' | head -1)
+            if [ -z "$weave_version" ]; then
+                weave_version=$(docker image ls | grep kurlsh/weave-npc | awk '{ print $2 }' | head -1)
+            fi
```
Comment (trelore) [inline scripts/tasks.sh:401]: Curious what @emosbaugh thinks about this, could `docker image ls` show both images (I'm thinking in cases of an upgrade)? If so, do we want to default to the kurlsh image as that should be the more 'up-to-date' on?

Comment (emosbaugh) [reply scripts/tasks.sh:401]: good point. agreed

Comment (garcialuis) [reply scripts/tasks.sh:401]: I swapped them, this is now checking for the kurlsh image first.

**Comment by garcialuis** [comment]
> you will need to add EXEC_IMAGE env var to pods like so https://github.com/replicatedhq/replicated-installer/blob/06a2cb9fc32731386e23afe87456b23386565610/install_scripts/templates/kubernetes/yml-generate.sh#L966-L967
> 
> can you find out if we have the ability to run airgap tests on pr from @laverya ? if so can you add? i would think we would have caught this without my review

@emosbaugh The env var and airgap test have been added now.

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**